regulament_pregatit_feedback.txt

```diff
@@ -535,54 +541,30 @@ Art. 87. Procedura de înființare a unui Centru Local este următoarea:
 		c. să plătească cotizația
 
 Art. 88.
-	(1) Centrul Local se consideră legal constituit în urma emiterii unei decizii de înființare, de către Consiliul Director, pe baza hotărârii Adunării Generale.
+	(1) Centrul Local se consideră legal constituit după înscrierea în Registrul asociațiilor și fundațiilor, în baza deciziei de înființare, emise de către Consiliul Director, pe baza hotărârii Adunării Generale.
 	(2) Condițiile necesare de îndeplinit sunt următoarele:
-		a. să cuprindă cel puțin două unități (25 - 30 membri)
-		b. să fi dovedit capacitate organizatorică și de inițiativă, pentru a fi integrat în structura ONCR.
-	(3) Centrele Locale astfel înființate nu au personalitate juridică și sunt sucursale.
-
-Art. 89. 
-	(1) Centrele Locale pot obține personalitate juridică (devenind filiale), dacă îndeplinesc următoarele condiții:
-		a. Centrul Local are decizie de înființare de cel puțin un an; se acceptă diferențe de până la cel mult o lună în cazul în care, din cauza planificării Adunărilor Generale ordinare în diferiți ani, perioada dintre Adunările Generale ordinare nu permite împlinirea unui an întreg.
-		b. are un număr de cel puțin 10 lideri și 50 de cercetași.
-		c. are un sediu stabil.
-		d. dovedește capacitate de gestiune financiară.
-		e. are putere financiară dovedită prin atragere de fonduri.
-		f. activul patrimonial, alcătuit din aportul în natură sau în bani al asociaților, este în valoare de cel puțin dublul salariului minim pe economie, la data respectivă.
-
-Art. 90. Personalitatea juridică permite Centrului Local angajarea de personal retribuit.
-        	
-Art. 91. Personalitatea juridică permite Centrului Local administrarea de centre scout sau de proiecte naționale pe un termen mai lung de un an.
-
-Art. 92. Centrele Locale cu personalitate juridică au obligația depunerii situației financiare anuale și a altor documente la administrația financiară de care aparțin, conform legilor în vigoare, precum și realizarea unui raport financiar anual (pe baza acestor documente) care este trimis Consiliului Director.
+		a. să cuprindă cel puțin două patrule (25 membri din care cel puțin 3 lideri)
+		b. să fi îndeplinit cel puțin criteriile minime pentru Nivelul 1 - Centre Locale în dezvoltare, în evaluarea IEDLOC.
+		c. în cazul Grupurilor externe care cer să devină Centre Locale este necesară recomandarea din partea Centrului Local Mentor.
+	(3) Centrele Locale astfel înființate au personalitate juridică. Filiala se constituie prin hotărârea Adunării Generale a ONCR. Personalitatea juridică se dobândește de la data înscrierii filialei în Registrul asociațiilor și fundațiilor.
+	(4) Toate Centrele Locale la înființare semnează un contract cu ONCR, prin care primesc susținerea organizației pentru a-și desfășura activitatea și a reprezenta local ONCR, drepturi care pot fi suspendate de Consiliul Director și retrase de Adunarea Generală.
```

Comment:
oare n-ar fi bine să intrăm mai în detaliu cu contractul ăsta?

Reply:
La ce te gândești când zici că mai în detaliu? să menționăm în regulament ce ar trebui să conțină contractul? Eu mă gândeam că venim cu o anexa cu model standard de contract.

Reply:
N-aș pune contractul ca anexă la Regulament, că asta complică mult schimbarea lui. Mai repede aș lista cumva elementele minime pe care contractul ar trebui să le conțină, și eventual lucrurile pe care contractul nu are voie să le conțină, fără să limitez prea mult capacitatea de reacție a CD la realitățile cu care o să se confrunte going forward.

Reply:
Contractul: nume și logo-ul entității, drepturile și obligațiile (dreptul de a participa la activități, obligația de plată a cotizației etc), reprezentarea cercetășiei (misiunea, viziune, valori și branding/reguli etc), angajament față de formare, angajament față de dezvoltarea comunității, respectarea Regulamentului, Statutului și deciziilor AG și CD, condițiile în care contractul este anulat (rezilierea contractului) etc

Reply:
Contractul (charter-ul, mandatul) nu ar trebui să se lege de persoane (nu apar nume de oameni, ci se leagă de structură);